src/pykeen/stoppers/early_stopping.py

```diff
@@ -152,7 +152,7 @@ def should_stop(self) -> bool:
 
         self.result_tracker.log_metrics(
             metrics=metric_results.to_flat_dict(),
-            step=self.number_evaluations,
+            step=(self.number_evaluations * self.frequency) + 1,
```

Comment:
does this formula appear multiple places in the EarlyStopper class? maybe you can make this a property